src/styles/variables.scss

```diff
@@ -0,0 +1,11 @@
+$size: 100px;
+$gap: 1px;
+$width: ($size * 7) + ($gap * 6);
+
+$days:
+  "tue" ($size+$gap),
+  "wed" 2 * ($size+$gap),
+  "thu" 3 * ($size+$gap),
+  "fri" 4 * ($size+$gap),
+  "sat" 5 * ($size+$gap),
+  "sun" 6 * ($size+$gap);
```

Comment:
Let's store only day index here, no need to duplicate this formula, use it only once